org.ekstep.questionunit.mcq-1.3/renderer/plugin.js

```diff
@@ -115,9 +115,11 @@ org.ekstep.questionunitmcq.RendererPlugin = org.ekstep.contentrenderer.questionU
     var resValues = [];
     var selectedIndex = MCQController.pluginInstance._selectedIndex;
     var value = {};
-    if (!_.isUndefined(selectedIndex))
+    if (!_.isUndefined(selectedIndex)){
```

Comment:
revert this file. There is no changes except indentation.

Reply:
@vinukumar-vs I have added if block {}